### PR TITLE
Feat/vb masters import

### PR DESCRIPTION
**Pull request by fredrikved**

### CODE REVIEW

File: workflows/ingest/vb_master.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	if len(files) > 1 {
+		return nil, fmt.Errorf("too many files in directory: %s", params.Directory)
+	}
+
+	outputDir, err := wfutils.GetWorkflowOutputFolder(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	//Production/aux/{date}/{wfID}/{filename}
```
Comment (KillerX) [inline workflows/ingest/vb_master.go:56]: Should this be aux?

**Review by KillerX — APPROVED** [review]
I think it looks good, just not sure about the aux path.

**Review by KillerX — APPROVED** [review]
I think it looks good, just not sure about the aux path.

File: workflows/ingest/vb_master.go (hunk 1 above)
Comment (fredrikved) [reply workflows/ingest/vb_master.go:56]: Switching it to raw.

**Pull request merged**